jgrapht-core/src/main/java/org/jgrapht/alg/isomorphism/GraphOrdering.java

```diff
@@ -161,21 +167,19 @@ public int getVertexCount()
      */
     public boolean hasEdge(int v1Number, int v2Number)
     {
-        V v1, v2;
-        Boolean containsEdge = null;
-
+        
         if (cacheEdges) {
-            containsEdge = adjMatrix[v1Number][v2Number];
+            final byte cache = adjMatrix[v1Number*vertexCount+v2Number];
```

Comment:
Save the address computation for use in the update below.

Reply:
done